googlesource/source_intergration_test.go

```diff
@@ -29,18 +31,15 @@ import (
 )
 
 var (
-	serviceAccount = "<replace_me>"       // replace with path to service account with permission for the project
-	projectID      = "conduit-connectors" // replace projectID created
-	datasetID      = "conduit_test_dataset"
-	tableID        = "conduit_test_table"
-	tableID2       = "conduit_test_table_2"
-	location       = "US"
+	serviceAccount   = os.Getenv("SERVICE_ACCOUNT") // eg, export SERVICE_ACCOUNT = "path_to_file"
```

Comment:
I suggest a bit more detailed description of what service account is. When I read, I think of a service account ID (e.g. an email), but it's actually a file. It would be good to describe what file that needs to be, how to generate it etc.

Reply:
added the description in Readme and spec file